securedrop/tests/functional/functional_test.py

```diff
@@ -261,6 +263,11 @@ def sd_servers(self):
 
                     self.admin_user["totp"] = pyotp.TOTP(self.admin_user["secret"])
 
+                    if self.journalist_designation_collision_test:
```

Comment:
We shouldn't introduce test-specific customization in this class. Instead, you could create the source app in a new method, and override that method in a derived class to limit the nouns and adjectives. Then `TestSourceInterfaceDesignationCollision` would use that subclass instead of the base `FunctionalTest`.